test/end-to-end/purchases/registry.spec.js

```diff
@@ -0,0 +1,13 @@
+const helpers = require('../shared/helpers');
+
+const PurchaseOrderSearch = require('./registry.search');
+
+// purchase order registry tests
+describe('Purchase Order Registry', function () {
+  'use strict';
+
+  before(() => helpers.navigate('#/purchases/list'));
```

Comment:
It would be nice to make this URL `/purchases`.  That way it would match the other registries:
 1. `/patients` -> patient registry
 2. `/payments` -> cash payments registry
 3. `/invoices` -> invoice registry.